keyboard/test/kb_strategy_spec.rb

```diff
@@ -54,8 +54,16 @@
         expect(kb_strategy).to receive(:write_udev_rule)
 
         kb_strategy.set_layout("es")
-      end      
+      end
+    end
 
-    end    
+    context "empty keyboard_code parameter" do
+      it "returns only " do
```

Comment:
NP: "returns only" sounds strange. I'd say something like "does not try to set the keyboard layout".